dartagnan/src/main/java/com/dat3m/dartagnan/parsers/program/visitors/VisitorSpirv.java

```diff
@@ -6,16 +6,15 @@
 import com.dat3m.dartagnan.parsers.program.visitors.spirv.*;
 import com.dat3m.dartagnan.program.Program;
 import com.dat3m.dartagnan.program.Register;
+import com.dat3m.dartagnan.program.specification.AbstractAssert;
 import org.antlr.v4.runtime.tree.ParseTree;
 
 import java.lang.reflect.Constructor;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
 
-public class VisitorSpirv extends SpirvBaseVisitor<Program> {
```

Comment:
It was because I wanted the visitSpirvOutput() to return AbstractAssert. 
Now I think it's better to move the aggregateAssertions() to ProgramBuilderSpv, I will change this back.